source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -275,31 +275,32 @@ def _get_duration(resp, place, mode, **kwargs):
     return map_response[resp.routing_status]
 
 
-def _sn_routing_matrix(instance, place, places_crowfly, mode, max_duration_to_pt, request, **kwargs):
+def _sn_routing_matrix(instance, origins, destinations, is_orig_center, mode, max_duration_to_pt, request, **kwargs):
```

Comment:
agreed (and also as you say, it's not the point of this PR, just a side effect when needed)